scripts/strict-null-checks-ci.js

```diff
@@ -0,0 +1,170 @@
+/**
+ * A script for formatting a Markdown report for CI on the number of strictNullChecks errors we're
+ * seeing on the current branch vs on main. The report also includes some info about the most
+ * error-filled files, as well as the errors we see most often.
+ */
+const fs = require('fs');
+
+/**
+ * Load JSON data, formatted by `tsc-output-parser`
+ *
+ * In the GH Actions workflow we write two json files, one for the PR
+ * branch and one for main (so that we can compare them).
+ *
+ */
+const prData = JSON.parse(String(fs.readFileSync('./null_errors_pr.json')));
```

Comment:
Should we wrap the `fs.readFileSync` calls here in a try/catch in the event that reading the file(s) fails? I suppose that if we fail to read the file, the workflow _should_ fail (and it's non-blocking on the PR landing).  I'm just not sure what the DX is like if we hit a failure here.  Do you know if the stacktrace makes it clear enough which file read failed?

Reply:
👍 I'm OK with that/leaving it as is for now in this case